graph-metrics-exporter/base/cronjob.yaml

```
@@ -0,0 +1,94 @@
+---
+kind: CronJob
+apiVersion: batch/v1beta1
+metadata:
+  name: graph-metrics-exporter
+  labels:
+    app: thoth
+    component: graph-metrics-exporter
+spec:
+  schedule: "@hourly"
+  suspend: true
+  concurrencyPolicy: Forbid
+  successfulJobsHistoryLimit: 2
+  failedJobsHistoryLimit: 4
```

lets reduce these to more smaller value
```suggestion
  successfulJobsHistoryLimit: 1
  failedJobsHistoryLimit: 1
```